### DISCUSSION TITLE
On Custom Features and the dropped support

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#5927] [General]
A mix of voicing opinion, rant and etc.

## How we started
Chromie kind of started as a _blizzlike_ server with the intention to fix AzerothCore step-wise.
There were events and things taking place during droughts to keep the community busy

Then we approached raid content (UBRS included) and it became apparent (also for the blind-sighted) that some tuning for content would be necessary to
1. provide a bit of a challenge to players
2. make bosses last a bit longer for them to even progress through their phases (cause there is no bug testing if a boss dies in 5 seconds)
3. all while staying casual-friendly

Further, the idea was to make things accessible and offer means to make things happen
1. xfaction to allow easier grouping for dungeons and battlegrounds
2. custom arena queue
3. phasing of world bosses
4. hellfire, zanga and halaa event

## Part 2: Rant

1. obviously introducing xfaction would either mean a lot of additional effort required to fix or adjust things or having an exit strategy. Currently, i am not aware if there is a planned exit point but latest if we talk about wintergrasp / togc things could get complicated. Earlier, we already will face issues with arena teams. And already since introduction, we are riddled by the fact that you get pvp enabled by healing across the factions in your group, causing you to also be affected by the 7 second delay to get out of combat from pvp.
2. arena queue nice and good - i didnt use it yet and dont plan to
3. introduction of world boss phasing was controversial to say the least. but it seems, similar to the next point, being thrown out like a hit potato, looked at for 1 week and then not being fixed any further. currently, we dont have any ``.worldboss lock `` for tbc, have boss invisible bug uppon death / regrouping, heals being "blocked" or reduced
4. a event which announces one thing in server spam and does / work totally different from programming point of view. ever tried the ``.fun off`` which did never work and now is also announced to become directly unavailable once a event starts? there is no real check of balances, heck why even start a 0vs3 event or a 3vs12 as we have seen. not to mention that the event areas itself are more or less broken (halaa token range), Zanga flag

##  My hot take
some of the people use Chromiecraft as a test playground to dive into some programming adventures. the intentions are usually good but it seems like as soon as a "working" state is reached, things are being dropped and thats about it.
Or things are practically dropped onto the community in very either a short notice or without tasting the water beforehand.

## outlook
I didnt do the survey, but I hope that there were at least some generic/open questions in asking about what the playerbase misses or were improvements would be appreciated. given the community is large and hence there are different opinions. But, I atleast would like to see a bit more focus on making the default content enjoyable / working correctly, before adding custom things that are getting dropped like hot potates as google does...

**Comment by Nyeriah** [comment]
Hello,

This thread seems to originate from a false premise. I'll address some of the points raised here:

1) There is no plan to ever disable crossfaction and there is no foreseable reason why we would ever wish to do so at this time, thus there is no "exit plan" for that. The benefits of crossfaction (having access to more players to play with) outweight any possible gains from disabling it. We will solve the issues with the crossfaction raids once we get there.

2) World boss support wasn't dropped and I am still working on addressing the issues. However, as something used during most part of the vanilla phase without issues, it is yet upon to us to figure what is causing them now and not before. The command was updated last week. Unsure why it still doesn't work on the live server, will be addressed in the next update.

>some of the people use Chromiecraft as a test playground to dive into some programming adventures. the intentions are usually good but it seems like as soon as a "working" state is reached, things are being dropped and thats about it.
Or things are practically dropped onto the community in very either a short notice or without tasting the water beforehand.

All of the mentioned features here were proposed as solutions to concerns raised by the community and not really anyone's "adventures"

I hope this helps clearing things up.

Yours sincerely.

Nyeriah
